examples/demo_calibration.py

```diff
@@ -0,0 +1,27 @@
+""" Pulse length conversion
```

Comment:
this file looks good --  no further review

Reply:
note edit to correspond with argument of the converter being amplitude and deblank